enterprise/backend/src/metabase_enterprise/advanced_config/file/databases.clj

```diff
@@ -37,18 +37,27 @@
 
 (defn- init-from-config-file!
   [database]
-  ;; assert that we are able to connect to this Database. Otherwise, throw an Exception.
-  (driver.u/can-connect-with-details? (keyword (:engine database)) (:details database) :throw-exceptions)
-  (if-let [existing-database-id (t2/select-one-pk Database :engine (:engine database), :name (:name database))]
+  (if (contains? database :delete)
```

Comment:
Can you add a small comment here explaining what all of this is for, similar to your PR description? It should include an explanation of why this feature needs to be in the config file, and not as a CLI command

Reply:
https://github.com/metabase/metabase/pull/46437/commits/b78dc9d4a6502ed19d32269ea63f0ec518caf418